ceph.spec.in

```diff
@@ -350,6 +350,7 @@ BuildRequires:	lz4-devel >= 1.7
 %if 0%{with make_check}
 %if 0%{?fedora} || 0%{?rhel}
 BuildRequires:	libtool-ltdl-devel
+BuildRequires:	ninja
```

Comment:
Failed for me on Fedora 33. Changing `ninja` to `ninja-build` fixed it.

Reply:
@rishabh-d-dave fixed and repushed.